velox/vector/arrow/tests/ArrowBridgeArrayTest.cpp

```diff
@@ -869,6 +869,54 @@ TEST_F(ArrowBridgeArrayExportTest, mapNested) {
   EXPECT_EQ(values.Value(1), 0);
 }
 
+TEST_F(ArrowBridgeArrayExportTest, mapTimestamp) {
+  const auto keys =
+      vectorMaker_.flatVector<int32_t>(5, [](vector_size_t i) { return i; });
+  auto values = vectorMaker_.flatVector<Timestamp>(
+      5, [](vector_size_t row) { return Timestamp(row, row); });
+  const auto offsets = makeBuffer<vector_size_t>({1, 4, 2});
```

Comment:
The code for fixing bug looks good to me. But I'm little confused by the test code. What's the purpose of building the MapVector without using all keys and values? Is it necessary for bug reproduce?

Reply:
The offset 4 exceeds [0, 2] when rows.count is 3, so it is a typical buggy case. There are also some similar tests using less keys/values in the MapVector.
https://github.com/facebookincubator/velox/blob/main/velox/vector/arrow/tests/ArrowBridgeArrayTest.cpp#L831-L834.

Reply:
Got it, thanks for your explanation.